.github/workflows/main.yml

```diff
@@ -146,7 +146,7 @@ jobs:
       - name: Install extra kernel modules(e.g. vrf)
         run: |
           sudo apt-get update;
-          sudo apt-get -y install "linux-modules-extra-$(uname -r)"
+          sudo apt-get -y install "linux-modules-extra-azure"
```

Comment:
The reason we use `uname -r` here is because repo might hold newer version than running kernel.

The CI failure should be fixed by https://github.com/actions/runner-images/issues/8080

Reply:
Sure, let's wait then. I am dropping the commit. Thanks for sharing the link.